lib/ansiblelint/utils.py

```diff
@@ -763,15 +763,48 @@ def is_playbook(filename):
     return False
 
 
+def get_yaml_files(options=None):
+    """Find all yaml files."""
+    if options is None:
+        options = {}
+
+    # git is preferred as it also considers .gitignore
+    git_command = ['git', 'ls-files', '*.yaml', '*.yml']
+    if options.verbosity:
+        print(
+            "Discovering files to lint: %s" % (' '.join(git_command))
+        )
+
+    try:
+        out = subprocess.check_output(
+            git_command,
+            stderr=subprocess.STDOUT,
+            universal_newlines=True
+        ).split()
+    except subprocess.CalledProcessError as exc:
+        if options.verbosity:
+            print(
+                "Warning: Failed to discover yaml files to lint using git: %s" % (
```

Comment:
Let's use `str.format`
```suggestion
                'Warning: Failed to discover yaml files to lint using git: {git_out!s}'.
                format(git_out=exc.output.rstrip('\n'))
```